### PR TITLE
Created item-info tab and populating claim & difficulty.

### PR DESCRIPTION
**Pull request by bgsavage**
I'm going to have quite a few changes to populate each of these data points, so I'm going to break it up as I implement each field.  In this case, it's just claim & difficulty which is data we already have loaded from the examitems call.  Other fields will require updates to the examitems call to get the needed data.

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<div>
+  <p>
+    <b>{{ 'labels.assessments.items.tabs.item-info.claim' | translate}}</b>
+    {{ 'definition.claim.' + item.claim + '.description' | translate }}
+  </p>
+  <p class="green">
+    // TODO:
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.html]: Lots of todos here.

File: webapp/src/main/webapp/src/app/student/responses/student-responses.component.html [hunk 2]
```diff
@@ -40,7 +40,7 @@ <h3 class="blue mb-md">{{'labels.student.responses.title' | translate}}</h3>
                     content="{{'labels.groups.results.assessment.items.cols.claim-info' | translate}}"></span>
       </ng-template>
       <ng-template let-item="rowData.assessmentItem" pTemplate="body">
-        {{ 'enum.claim.' + item.claim | translate }} / {{ 'labels.groups.results.assessment.items.target' | translate:item }}
+        {{ 'definition.claim.' + item.claim + '.name' | translate }} / {{ 'labels.groups.results.assessment.items.target' | translate:item }}
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/app/student/responses/student-responses.component.html]: moved the json location as it's more than just enum now.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 3]
```diff
@@ -357,7 +367,7 @@
       "3": "Met Standard",
       "4": "Exceeded Standard"
     },
-    "claim-code": {
+    "subject-claim-code": {
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: renamed for clarity

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 4]
```diff
@@ -388,25 +398,60 @@
...
       "E": "Easy",
       "M": "Moderate",
       "D": "Difficult"
     }
   },
+  "definition": {
+    "claim": {
+      "1-IT": {
+        "name":"Read Analytically: Informational Text",
+        "description": "Read Analytically: Informational Text - Students can read closely and analytically to comprehend a range of increasingly complex literary and informational texts."
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: If the table is small enough, we will load the description in the en.json.  For common core standard & target, I'll make individual api endpoints to get that data that at some future point we can run through translate (but for now will just get the description directly)

File: webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.html (hunk 1 above)
Comment (esotrope) [inline webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.html]: Is it green so it stands out as a todo or will it actually be green? : j

Comment (bgsavage) [inline webapp/src/main/webapp/src/app/assessments/items/item-info/item-info.component.html]: As a todo so it looks like a comment if you open the tab :) . It'll help me keep track and let any testers know it's not done.

image

**Pull request merged**